.github/workflows/release.yml

```diff
@@ -61,6 +61,9 @@ jobs:
       - name: Building packages
         run: yarn build
 
+      - name: Building package READMEs
+        run: yarn generate-readmes
```

Comment:
I removed this from the `prebuild` command, therefore we need to run this before we release the packages.